src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -133,6 +137,12 @@ body {
     margin-right: .5em;
 }
 
+.separate-content-holder {
+    height: 75px;
```

Comment:
It's better to use `em` or `rem` instead of `px`, see: [Why em instead of px](http://stackoverflow.com/a/609561/4819795)